extensions/2.0/Khronos/KHR_accessor_float64/README.md

```diff
@@ -0,0 +1,130 @@
+<!--
+Copyright 2024 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR\_accessor\_float64
+
+## Contributors
+
+- Alexey Knyazev, [@lexaknyazev](https://github.com/lexaknyazev)
+- Sean, [@spnda](https://github.com/spnda)
+
+Copyright 2024 The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 Specification.
+
+## Overview
+
+The base glTF 2.0 Specification supports only one floating-point component type - IEEE-754 _binary32_ (also known as single-precision). Although this is sufficient for most applications, there are use cases that benefit from increased range and precision provided by the 64-bit floating-point data type (also known as double-precision).
+
+This extension expands the set of recognized accessor component types with a new enum value `5130` that represents IEEE-754 _binary64_.
+
+This extension allows usage of such accessors in all places where the single-precision floating-point component type is accepted by the base glTF 2.0 Specification or extensions.
+
+Because this extension does not provide a way to specify alternative data representations, assets that use this extension **MUST** specify it in the `extensionsRequired` array.
+
+## Extending Accessors
+
+When the extension is supported, the following data type is valid for accessors in addition to the types defined in [Section 3.6.2.2](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#accessor-data-types).
+
+| `componentType` | Data Type | Signed | Bits |
+|-----------------|-----------|--------|------|
+| `5130`          | _double_  | Signed | 64   |
```

Comment:
Out of curiosity, does `5130` come from anywhere in WebGL, or is it unique to glTF?

Reply:
It comes from desktop OpenGL (like all other component type enums originally).

Reply:
Of course. Thanks!

Reply:
Not sure what "THE" official source is, but https://registry.khronos.org/OpenGL/api/GL/glcorearb.h says
`#define GL_DOUBLE                         0x140A`
(and it's found in many other places via https://github.com/search?q=org%3AKhronosGroup+GL_DOUBLE&type=code

(I'm usually using https://javagl.github.io/GLConstantsTranslator/GLConstantsTranslator.html to check stuff like that, but... I don't even remember what header file I originally used for that 8 years ago - it may be incomplete...)